cmd/security-secretstore-setup/Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN go mod download
 
 COPY . .
 
-RUN make cmd/security-file-token-provider/security-file-token-provider cmd/security-secretstore-setup/security-secretstore-setup
+RUN make cmd/security-file-token-provider/security-file-token-provider \
+  cmd/security-secretstore-setup/security-secretstore-setup\
```

Comment:
Nit pick: for consistence sake, may leave empty space at the end before \

Reply:
Updated and push'ed